packages/contracts-periphery/contracts/universal/op-nft/AttestationStation.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Semver } from "@eth-optimism/contracts-bedrock/contracts/universal/Semver.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+/**
+ * @title AttestationStation
+ * @dev   Contract for creating attestations.
+ * @notice This contract is not yet audited.
+ */
+contract AttestationStation is Semver {
+    /**
+     * @notice Struct representing data that is being attested
+     *
+     * @custom:field about                    Address being attested about (not creator/msg.sender)
+     * @custom:field key                      A bytes32 key for the attestation.
+     * @custom:field val                      The attestation as arbitrary bytes
```

Comment:
I copied this comment originally from contracts-bedrock.   Should it be @custom?  What does @custom mean?

Reply:
Also nitpick but the descriptions should be separated by one space from the longest field name